fineract-provider/src/main/java/org/apache/fineract/ServerApplication.java

```diff
@@ -54,6 +54,7 @@ private static SpringApplicationBuilder configureApplication(SpringApplicationBu
     }
 
     public static void main(String[] args) throws IOException {
+        System.setProperty("liquibase.analytics.enabled", "false");
```

Comment:
wouldn't it be placed more strategically at the TenantDatabaseUpgradeService itself, in a static initializer? Can you test it if it works the same way there?

Reply:
@meonkeys I think it can be placed as a Fineract Property set to false by default and then this property, (instead of having the System Property), can be placed in the TenantDatabaseUpgradeService as suggested by @kjozsa in the TenantDatabaseUpgradeService as static initializer

Reply:
@kjozsa: yes, it does.

@kjozsa and @IOhacker: how about we go with just this simple static initializer in TenantDatabaseUpgradeService:

```java
static {
    System.setProperty("liquibase.analytics.enabled", "false");
}
```

I can't imagine anyone *wanting* to set the flag to `true`, and it would be many fewer lines of code just to hardcode it to `"false"` in this manner rather than provide a configurable setting. If there's demand to make it configurable later, we could of course do that. I feel like our use of Liquibase is up to us, though. An under the hood / upstream decision we get to make. Thoughts?

Also, is it clear that we *must* use `System.setProperty`? Our only other alternative is to use an environment setting `LIQUIBASE_ANALYTICS_ENABLED=false`, visible to the Java VM before/as it initializes. I don't see any way in the Liquibase API to disable phoning home at runtime.